force-app/classes/src/MethodSpy.cls

```diff
@@ -18,32 +18,63 @@
 @IsTest
 public class MethodSpy {
   public String methodName;
-  private Integer callCount = 0;
-  private List<Object> lastCallParams;
+  private CallLog callLog;
   private List<ParameterizedMethodSpyCall> parameterizedMethodCalls = new List<ParameterizedMethodSpyCall>();
   private Object returnValue;
   private Boolean configuredGlobalReturn = false;
   private Exception exceptionToThrow;
 
   public MethodSpy(String methodName) {
     this.methodName = methodName;
+    this.callLog = new CallLog();
   }
 
-  public List<Object> getLastCallParams() {
```

Comment:
**discussions:** I don't think we should keep that method now @LudoMeurillon 

Because we don't need it anymore with the new APIs

But let's discuss and track our decision here